flink-connectors/flink-connector-hive/src/main/java/org/apache/flink/connectors/hive/HiveTablePartition.java

```diff
@@ -56,4 +56,14 @@ public StorageDescriptor getStorageDescriptor() {
 	public Map<String, Object> getPartitionSpec() {
 		return partitionSpec;
 	}
+
+	public static boolean isParquet(StorageDescriptor sd) {
+		return sd.getSerdeInfo().getSerializationLib()
+				.toLowerCase().contains("parquet");
+	}
+
+	public static boolean isOrc(StorageDescriptor sd) {
+		return sd.getSerdeInfo().getSerializationLib()
+				.toLowerCase().contains("parquet");
```

Comment:
maybe a word mistake? it is orc instead of parquet

Reply:
Yes misstake